modules/standard/Map.chpl

```diff
@@ -450,11 +450,19 @@ module Map {
     /*
       Iterates over the key-value pairs of this map.
 
-      :yields: A tuple of references to one of the key-value pairs contained in
-               this map.
+      :yields: A tuple containing a copy of one of the key-value pairs
+               contained in this map.
```

Comment:
Can we avoid repeating "contain" here? "A tuple with copy of" ? "A tuple that has a copy of" ?

Reply:
Thanks!